docker/DockerBuild.sh

```diff
@@ -57,3 +58,5 @@ DOCKER_COMMAND="docker build --build-arg UID=$UID_ARG \
                              apacheminificpp:$MINIFI_VERSION ."
 echo "Docker Command: '$DOCKER_COMMAND'"
 ${DOCKER_COMMAND}
+
+rm -rf $CMAKE_SOURCE_DIR/docker/minificppsource
```

Comment:
What's the intent of doing it twice? 
I'm fine with either doing cleanup at the start of the build or at end end, but don't get the point of doing both.